### DISCUSSION TITLE
SMTP-IP Missing Key section

### DISCUSSION THREAD

**Discussion by mryanbc** [postalserver/postal#3072] [Help with using Postal] [answered]
In the SMTP-IP section there is no Key generated, or one that can be seen to be generated.

![image](https://github.com/user-attachments/assets/ffd1282e-d3a9-4c26-82db-8219a49613bf)

If I just do the SMTP-IP setting, then on Credentials, Password field is empty. 

Postal 3.3.4

**Comment by willpower232** [comment]
The point of SMTP-IP is that Postal is trusting this device by its IP address so a password is not required.

If you would like a password then you can use the SMTP type instead.

**Reply by mryanbc** [reply]
I see, however when I set it, Postal is saying not being used. Maybe I am not setting correctly. When I want an SMTP application to connect to Postal. What password should I set when using SMTP IP? As if i only set up SMTP IP (not SMTP), there is no password to use on the SMTP applications. Maybe I am not understanding what to do.

**Reply by willpower232** [reply, marked as answer]
SMTP-IP is mostly for relaying purposes, if you're trying to send email normally then you just need a regular SMTP type credential.

You could try looking at `postal logs smtp` to see what is actually happening when you're trying to send a message.
